### PR TITLE
Removes an unused var on /mob/living

### PR DESCRIPTION
**Pull request by DGamerL**
## What Does This PR Do
Removes an unused variable on `/mob/living`

## Why It's Good For The Game
It's quite stupid to load a var on `/mob/living` and to never remove it if it isn't used anymore.

## Testing
Uhhhhhhh

## Changelog
NPFC

### CODE REVIEW

**Review by Henri215 — APPROVED** [review]
LGTM (Let's Gamble, Try Merging)

**Review by Contrabang — APPROVED** [review]
LGTM (Let's go to Morocco)

**Review by lewcc — APPROVED** [review]
LGTM (Lose game time. man)

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -1,5 +1,4 @@
 /mob/living
-	var/canEnterVentWith = "/obj/item/implant=0&/obj/item/clothing/mask/facehugger=0&/obj/item/radio/borg=0&/obj/machinery/camera=0"
 	var/datum/middleClickOverride/middleClickOverride = null
```
Comment (hal9000PR) [inline code/modules/mob/living/carbon/carbon.dm:2]: ok, I know this isnt your fault, but can you PLEASE move these other vars to `living_defines.dm` i HATE that we have these random defines slapped here in carbon.dm and it breaks VSC "go to definition" macros

**Review by SteelSlayer — APPROVED** [review]

**Pull request merged**